data/json/recipes/other/materials.json

```diff
@@ -32,8 +32,7 @@
       [ [ "forge", 80 ], [ "char_forge", 80 ] ]
     ],
     "components": [
```

Comment:
If you're using an *nix system then go with the commands I posted. I would suspect most windows users would prefer double clicking a binary and waiting a few moments as opposed to manually checking a file?